README.md

```diff
@@ -958,6 +959,14 @@ Limitations:
 - Only direct descendants of a slide section can be stretched
 - Only one descendant per slide section can be stretched
 
+### Resize Event
+
+When reveal.js changes the scale of the slides it fires an resize event. You can subscribe to the event to resize your elements accordingly.
```

Comment:
`an` should be `a` here

Reply:
Good catch, just merged this and will fix the typo.